themes/src/main/resources/theme/keycloak.v2/login/resources/css/styles.css

```diff
@@ -109,6 +109,10 @@ div.kc-logo-text span {
     text-align: center;
 }
 
+#kc-code pre code {
+    word-break: break-all;
+}
```

Comment:
Rather than introducing custom CSS, I would prefer we use [PatternFly utility classes](https://v5-archive.patternfly.org/utility-classes/text#controlling-text) where possible.